qcodes/data/__init__.py

```diff
@@ -1,7 +0,0 @@
-from qcodes.data.data_set import DataSet, new_data, load_data
-from qcodes.data.location import FormatLocation
-from qcodes.data.data_array import DataArray
-from qcodes.data.format import Formatter
-from qcodes.data.gnuplot_format import GNUPlotFormat
-from qcodes.data.hdf5_format import HDF5Format
-from qcodes.data.io import DiskIO
```

Comment:
Why delete these? I think they are fine to stay

Reply:
reverted back